### DISCUSSION TITLE
Aiogram 3 bot The handler works just with If, in another way it says not handled. How to fix it?

### DISCUSSION THREAD

**Discussion by seleznevaalis123** [community/community#109487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="784" alt="Снимок экрана 2024-02-22 в 00 36 44" src="https://github.com/community/community/assets/89939174/f991bb8c-e978-4259-8e68-877267cc14cb">
<img width="1440" alt="Снимок экрана 2024-02-22 в 00 36 26" src="https://github.com/community/community/assets/89939174/6a7c6231-2807-4851-9a48-16dda2c0ec90">

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @seleznevaalis123! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by seleznevaalis123** [comment]
solved
